src/data/data_classes_utils.py

```diff
@@ -104,14 +112,23 @@ def make_masks(
 
                 for point_array in annotations[finding]['polygons']:
                     # for line-like findings we draw lines with default width of 15px
-                    if finding in ('Kerley', 'Cephalization'):
+                    if (
+                        finding in ('Kerley', 'Cephalization')
+                        and np.amax(point_array, axis=0)[0] < width
+                        and np.amax(point_array, axis=0)[1] < height
+                    ):
                         draw.line(
                             point_array.flatten().tolist(),
                             fill=0,
                             width=linelike_finding_width,
                         )
-                    else:
+                        drawn = True
+                    elif (
+                        np.amax(point_array, axis=0)[0] < width
+                        and np.amax(point_array, axis=0)[1] < height
+                    ):
                         draw.polygon(point_array.flatten().tolist(), fill=0, outline=0)
+                        drawn = True
```

Comment:
@mak-en 

I think these checks are not needed if you are initially dropping **lateral** cases from the dataframe.